api/src/main/webapp/static/initialize_notebook_runtime.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Log all commands, logs are written to the Leo staging directory during startup
+# and can be found via the Leo API's GetRuntimeResponse.asyncRuntimeFields.stagingBucket.
+set -x
```

Comment:
TOL: What do you think about include `-e` like `-ex`?

Reply:
It was a consideration, but in general we want to allow for fine grain control of when a failure in this script results in exiting non-zero. The net result of failing here is an opaque error on createRuntime, i.e. when you attempt to open a notebook. Arguably we should fail if wondershaper does not start correctly, but failure on the other commands would not be severe.